book/modules/appendix2.tex

```diff
@@ -68,8 +93,76 @@
 	\begin{example}
 		Which of the follow matrices are in reduced row echelon form? For those that are, identify which
 		columns are pivot columns. For those that are not, what condition(s) fail?
-
-		XXX Finish include examples of 4 matrices, one of which is in RREF
+		\[
+			A=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 2\\
+			0 & 0 & 0 & 0\\
+			0 & 0 & 1 & 7
+			\end{bmatrix}\qquad
+			B=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 8\\
+			0 & 1 & 3 & 7\\
+			0 & 2 & 1 & 4\\
+			\end{bmatrix}\qquad
+			C=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 2\\
+			0 & 1 & 0 & 1\\
+			0 & 0 & 1 & 8
+			\end{bmatrix}\qquad
+			D=
+			\begin{bmatrix}[cccc]
+			0 & 1 & 3 & 6\\
+			1 & 0 & 0 & 9\\
+			0 & 0 & 1 & 4\\
+			\end{bmatrix}
+		\]
+		
+		\[
+			A=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 2\\
+			{\color{mypink} 0} & {\color{mypink} 0} & {\color{mypink} 0} & {\color{mypink} 0}\\
+			0 & 0 & 1 & 7
+			\end{bmatrix}
+		\]
+		
+		The second row of $A$ is a row of zeros and it does not occur at the bottom, so the fourth condition fails and $A$ is not in reduced row echelon form.
```

Comment:
Put the explanation before the appearance of the matrix. Since the conditions aren't numbered, refer to them by property only. You can write `The second row of $A$ is a row of zeros but does not occur at the bottom, so $A$ is not in reduced row echelon form.`